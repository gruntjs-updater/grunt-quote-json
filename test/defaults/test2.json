{
  "string": "foo",
  "numeric": 1234,
  "nochange": "NOCHANGE",
  "object": {
    "key": "value"
  },
  "array": [
      "item"
  ],
  "children": {
    "boy": "boy",
    "girl": "girl"
  }
}
