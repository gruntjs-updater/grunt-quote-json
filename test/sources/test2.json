{
  "string": "FOO",
  "numeric": 5678,
  "nocopy": "NOCOPY",
  "object": {
    "key": "VALUE"
  },
  "array": [
      "ITEM"
  ],
  "children": {
    "boy": "BOY",
    "baby": "BABY"
  }
}
