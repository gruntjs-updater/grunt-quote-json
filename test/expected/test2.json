{
  "string": "FOO",
  "numeric": 5678,
  "nochange": "NOCHANGE",
  "object": {
    "key": "VALUE"
  },
  "array": [
      "ITEM"
  ],
  "children": {
    "boy": "boy",
    "girl": "girl"
  }
}
